src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => <>Put the user here</>;
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-undef */
+/* eslint-disable react/jsx-no-useless-fragment */
```

Comment:
Consider removing unnecessary ESLint disable comments if they are not needed. This will help keep the code clean and maintainable.